udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -87,12 +118,7 @@ private void removeOverlapsWithTimeRange(
     for (TimeRange currentTimeRange : rangesSnapshot) {
 
       // Check for overlap and adjust currentTimeRange
-      if (currentTimeRange.overlaps(possiblyOverlappingTimeRange)) {
-        timeRanges.remove(currentTimeRange);
-        replaceOverlappedTimeRange(possiblyOverlappingTimeRange, currentTimeRange, timeRanges);
-      }
-
-      if (possiblyOverlappingTimeRange.overlaps(currentTimeRange)) {
+      if ((currentTimeRange.overlaps(possiblyOverlappingTimeRange)) || (possiblyOverlappingTimeRange.overlaps(currentTimeRange))) {
```

Comment:
Nit: Remove extra parenthesis around currentTimeRange.overlaps call and possiblyOverlappingTimeRange.overlaps call.